packages/product-components/src/components/SelectAssetModal/SelectAssetModal.tsx

```diff
@@ -125,14 +129,15 @@ export const SelectAssetModal = ({
             ? item.coingeckoId === searchCategory.coingeckoId ||
               item.coingeckoId === searchCategory.coingeckoNativeId
             : true;
+        const networkContractAddress = `${item.networkSymbol}${item.contractAddress ? `--${item.contractAddress}` : ''}`;
```

Comment:
I assume this refers to support for cryptoId.

Reply:
Yes, this is there because before it was searching by cryptoId (cryptoId is specific to coinmarket and has this structure of network--contract), now it's no longer there, because in my case I want to disable search by network, so I recreated it this way in case we search by network will be enabled